wsgi.py

```diff
@@ -1,14 +1,25 @@
 from application import create_app
 from application.admin import create_admin
-import click
+from flask_limiter import Limiter
+from flask_limiter.util import get_remote_address
 
+import click
 app = create_app()
 
+limiter = Limiter(
+    app=app,
+    key_func=get_remote_address,
+    default_limits=["1000 per day", "100 per hour"]
```

Comment:
Is the limit stated here of a user or for multiple users?

Reply:
It's the limit per unique IP address which should represent a unique user for now.